db_facts/db_info.py

```diff
@@ -69,6 +71,7 @@ def db(db_name: DBName, dbcli_config: DBCLIConfig = None) -> DBFacts:
                 dbcli_config['exports_from'][method]['pull_lastpass_from']
             lastpass_entry_name = template(template_for_lastpass_entry_name,
                                            (db_info, {}))
+            lastpass_entry_name = translate_secret_id_to_sm(lastpass_entry_name)
```

Comment:
mentioned this to you in a DM but posting here so we don't forget - looks like this is converting ALL db-facts entries to secretsmanager ones, which isn't what we always want to do.